data/tlg0062/tlg001/tlg0062.tlg001.perseus-eng1.xml

```diff
@@ -0,0 +1,683 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="eng">Phalaris</title>
+<author>Lucian of Samosata</author>
+<editor>Austin Morris Harmon, 1878-1950</editor>
+<funder>National Endowment for the Humanities</funder>
+<principal>Gregory Crane</principal>
+<respStmt>
+<persName>Gregory Crane</persName>
+<resp>OCRd, tagged and added initial corrections to  the text</resp>
```

Comment:
Please look at the Greek for the correct info here. There should be the other contributors here based on the staff timelines, which I have, and the creation dates. You are describing work that belongs in the change log.

Reply:
Note this incomplete: We use the original release date, not the update year. Again, I have this info

```
<publicationStmt>
<publisher>Trustees of Tufts University</publisher>
<pubPlace>Medford, MA</pubPlace>
<authority>Perseus Project</authority>
 <date type="release" when="2010-12-13">Perseus 4.0</date>
```

Reply:
`<resp>` has limited use for giving credit (Digital proofreader, etc.)